src/components/ChoiceCollectionSelectInput/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { useMemo, useState } from 'react';
+import { isNotDefined } from '@togglecorp/fujs';
+import { gql, useQuery } from '@apollo/client';
+import { SearchSelectInput } from '@the-deep/deep-ui';
+
+import {
+    ChoiceCollectionsQuery,
+    ChoiceCollectionsQueryVariables,
+} from '#generated/types';
+
+const CHOICE_COLLECTIONS = gql`
+    query ChoiceCollections(
+        $projectId: ID!,
+        $questionnaireId: ID!,
+        $search:String
+        ) {
+    private {
+        projectScope(pk: $projectId) {
+            id
+            choiceCollections(
+                filters: {
+                    questionnaire: {pk: $questionnaireId},
+                    name: {iContains: $search }
```

Comment:
I think if we want to filter the choiceCollections, we should do it through `label` instead of `name`.